test/toy_object_test.rb

```diff
@@ -70,13 +74,24 @@
       it "returns value of instance variable and unsets it" do
         call_method(:instance_variable_set, :@bar, :foo)
         assert call_method(:instance_variable_defined?, :@bar)
-    
+
         assert_equal :foo, call_method(:remove_instance_variable, :@bar)
         assert_nil call_method(:instance_variable_get, :@foo)
         refute call_method(:instance_variable_defined?, :@foo)
       end
     end
 
+    describe "getting an instance's class" do
+      specify "returns the class" do
+        my_class = class_klass.new
+        name = my_class.is_a?(ToyClass) ? "toy_new" : "new"
+        my_object = my_class.public_send(name)
+
+        name = my_object.is_a?(ToyObject) ? "toy_class" : "class"
+        assert_equal my_class, my_object.public_send(name)
```

Comment:
In the cold light of day, this test doesn’t really make sense here — it’s not using `klass` or `@object` at all, so we just end up testing `Class` and `ToyClass` three times each. It did help us to make progress but maybe we should remove it to avoid confusion.

Reply:
But isn’t that what the [next test](https://github.com/adrianna-chang-shopify/ruby-object-model/pull/7/files#diff-56590903b5dfe6dd73c803cae6fcc202dc299f66d9066706b4a343d3e1f8f9b7R95-R97) tests?

Reply:
I was just about to reply that I'd realized that 😂

Reply:
Sorry, it's been a hot minute since I've looked at this code 😂

Reply:
Teamwork!